### PR TITLE
Added saving log headers

### PR DESCRIPTION
**Pull request by afgTheCat**
Hey,

I don't think there is a way to save the log headers currently. This pr introduces an option to save them. I use this option to work with my logs, lmk if there is a need for this feature/if I should change anything :)

### CODE REVIEW

**Comment by haslinghuis** [comment]
@afgTheCat please rebase

**Comment by afgTheCat** [comment]
@haslinghuis synced with the github tool

File: src/blackbox_decode.c [hunk 1]
```diff
@@ -1073,6 +1075,50 @@ void resetParseState() {
...
+    if (*lineStart != 'H') {
+        return;
+    }
+
+    if (*(lineStart + 1) != ' ') {
+        return;
+    }
+    const char *separatorPos = strchr(lineStart, ':');
+    if (separatorPos == NULL || separatorPos > lineEnd) {
+      return;
```
Comment (haslinghuis) [inline src/blackbox_decode.c:1092]: Please use indentation of 4 consistently (only showing this once)
```suggestion
        return;
```

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
- approving untested, only visual review.

**Pull request merged**